ontotext-graphql-playground-component/src/mappers/fetcher-configuration.mapper.ts

```diff
@@ -0,0 +1,26 @@
+import {ExternalGraphqlPlaygroundConfiguration} from '../models/external-graphql-playground-configuration';
+import {Fetcher} from '../graphiql-models/fetcher';
+
+/**
+ * Utility class for converting an external GraphQL playground configuration into a fetcher configuration compatible with GraphiQL.
+ */
+export class FetcherConfiguration {
```

Comment:
I don't like this class name. If it's a mapper, then call it a mapper. Right now it looks more like a model.